test/integration/schema/index.js

```diff
@@ -11,9 +11,11 @@ module.exports = function(knex) {
     describe('dropTable', function() {
 
       it('has a dropTableIfExists method', function() {
+        this.timeout(30000);
```

Comment:
(process.env.KNEX_TEST_TIMEOUT || 5000) + 30000 or Math.max (KNEX_TEST_TIMEOUT, 30000) would be better, because mocha timeout is actually set to 60 seconds in CI